kv/key.go

```diff
@@ -78,7 +78,25 @@ type KeyRange struct {
 
 // IsPoint checks if the key range represents a point.
 func (r *KeyRange) IsPoint() bool {
-	return bytes.Equal(r.StartKey.PrefixNext(), r.EndKey)
+	if len(r.StartKey) != len(r.EndKey) {
```

Comment:
Please add some comments in this function.

Reply:
The new code is very hacking, I think it's better add commit like this:

`// return bytes.Equal(r.StartKey.PrefixNext(), r.EndKey)`